src/BuildScriptGeneratorCli/Commands/BuildScriptGeneratorOptionsHelper.cs

```diff
@@ -53,8 +52,6 @@ internal static class BuildScriptGeneratorOptionsHelper
 
             options.ScriptOnly = scriptOnly;
 
-            options.EnableDynamicInstall = enableDynamicInstall;
```

Comment:
Since we removed it from command line options, we may need to remove it from entrypoint of appservice-build as well.

Reply:
Yes, right. Once we get a release build out we can update the tag and make the change to environment variable. Actually I feel we need this environment variable to be the default for the image `github-actions` since this image does not have any sdks at all and so does not make sense to have it optional.

Reply:
In the latest commit, I made installing dynamically 'true' by default for github-actions image

Reply:
Cool, that makes sense for the `github-actions`.